cmake/externalFunc.cmake

```diff
@@ -2,7 +2,7 @@ set( CMAKE_EXECUTE_PROCESS_COMMAND_ECHO STDOUT)
 
 macro(addExternalFolder NAME FOLDER )
     # External resources/repositories are downloaded and built at configuration stage
-    message(INFO "[addExternalFolder] process ${NAME} ${FOLDER}")
+    message("[addExternalFolder] process ${NAME} ${FOLDER}")
```

Comment:
```suggestion
    message(STATUS "[addExternalFolder] process ${NAME} ${FOLDER}")
```

Reply:
OK for that. But, according to the cmake message documentation (https://cmake.org/cmake/help/v3.13/command/message.html), thereis a difference between `none` and `STATUS` for the message type. Even more on current cmake version (https://cmake.org/cmake/help/latest/command/message.html).
This is clearly out of scope of the current PR/discussion but we perhaps have to be more coherent on our cmake `message` usage and identification.